product-updates/trace-2-release-note.md

```diff
@@ -0,0 +1,35 @@
+# SysKit Trace - A centralized solution for documenting Office 365 settings and tracking settings changes
+
+We have prepared a new version of SysKit Trace, and it comes with some important and exciting new features. SysKit Trace now fully supports accounts with multi-factor authentication enabled, and now has built-in automatization and alerting capabilities. Not yet a customer? Download a free trial.
+
+[Try SysKit Trace Free](https://www.syskit.com/products/trace/download/)
+
+**Product version:** 2.0.0  
+**Build number:** 581  
+**Release date:** Apr 29, 2020
+
+## Support for Multi-Factor Authentication
+
+Multi-factor authentication (MFA) is a standard way to make your users’ and organization’s data more secure. With this release, SysKit Trace supports MFA for global administrator accounts.

```

Comment:
users' should be user's

Reply:
"Users" is used in plural here?

Reply:
Not sure if it is plural, if it is then " ' " sign is not needed at all, if it is singular then it should switch place :D